packages/odyssey-react-mui/src/labs/SideNav/SideNav.tsx

```diff
@@ -21,31 +21,30 @@ import {
   useEffect,
 } from "react";
 
-import { ExpandLeftIcon } from "../../icons.generated";
 import { NavAccordion } from "../NavAccordion";
 import {
   DesignTokens,
   useOdysseyDesignTokens,
 } from "../../OdysseyDesignTokensContext";
 import type { SideNavProps } from "./types";
 import { OktaLogo } from "./OktaLogo";
+import { HandleIcon } from "./HandleIcon";
+import { CollapseIcon } from "./CollapseIcon";
 import { SideNavHeader } from "./SideNavHeader";
 import {
   SideNavItemContent,
   SideNavListItemContainer,
 } from "./SideNavItemContent";
 import { SideNavFooterContent } from "./SideNavFooterContent";
+import { SideNavItemContentContext } from "./SideNavItemContentContext";
 
 export const DEFAULT_SIDE_NAV_WIDTH = "300px";
 
-const SideNavContainer = styled("div", {
-  shouldForwardProp: (prop) => prop !== "expandedWidth",
-})(({ expandedWidth }: { expandedWidth: SideNavProps["expandedWidth"] }) => ({
+const SideNavContainer = styled.div({
```

Comment:
**MINOR NIT PICK**

```suggestion
const SideNavContainer = styled.div({
```
Any reason we changed from `styled("div", {`? I think the dot version is deprecated now right? It doesn't match the other uses in the code even the one right below this. I'd like these all to be the same if possible.